core/kazoo_number_manager/src/converters/knm_converters.erl

```diff
@@ -169,12 +172,14 @@ to_db(_) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec is_reconcilable(ne_binary()) -> boolean().
+is_reconcilable(<<"+", _/binary>> = _Number) -> 'true';
```

Comment:
Nope. This is very bad. Try `+blalbla` or `+123123+12`.
These are actual numbers that got into sandbox at some point.